libraries-data-mariadb4j/src/test/java/com/baeldung/data/mariadb4j/JavaLiveTest.java

```diff
@@ -0,0 +1,83 @@
+package com.baeldung.data.mariadb4j;
+
+import ch.vorburger.mariadb4j.DB;
+import ch.vorburger.mariadb4j.DBConfigurationBuilder;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class JavaLiveTest {
+    @Test
+    void whenStartingADatabase_thenTheDatabaseIsUsable() throws Exception {
+        DB db = DB.newEmbeddedDB(0);
+        try {
+            db.start();
+
+            assertEquals("jdbc:mariadb://localhost:" + db.getConfiguration().getPort() + "/testing",
+                db.getConfiguration().getURL("testing"));
+        } finally {
+            db.stop();
+        }
+    }
+
+    @Test
+    void whenStartingADatabaseWithConfiguration_thenTheDatabaseIsUsable() throws Exception {
+        DBConfigurationBuilder config = DBConfigurationBuilder.newBuilder();
+        config.setPort(13306);
+        config.setDataDir("/tmp/data");
+        config.setBaseDir("/tmp/db");
```

Comment:
Do not hard code a directory, use a unique temporary directory and clean it up after the test (or just don't use config.setDataDir and config.setBaseDir in this example)

Reply:
Dang. Good point :( Fixed.